octobot_evaluators/api/evaluators.py

```diff
@@ -43,30 +44,50 @@ async def create_evaluators(evaluator_parent_class,
                             matrix_id: str,
                             exchange_name: str,
                             bot_id: str,
-                            symbols_by_crypto_currencies: dict = None,
+                            symbols_by_crypto_currency_names: dict = None,
                             symbols: list = None,
                             time_frames: list = None,
                             relevant_evaluators=CONFIG_WILDCARD) -> list:
+    crypto_currency_name_by_crypto_currencies = {}
+    symbols_by_crypto_currency_tickers = {}
+    for name, symbol_list in symbols_by_crypto_currency_names.items():
+        if symbol_list:
+            ticker = split_symbol(symbol_list[0])[0]
```

Comment:
it's the currency ticker of the 1st symbol of the list yep (BTC for BTC/USDT)

Reply:
There is a specific method for this in exchange (`get_pair_cryptocurrency`),I don't think we should duplicate this code. It can also be a false cryptocurrency name with a simple split.